quick_select/quick_select.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+
```

Comment:
It would be nice to add comments about the parameters making it easy for a reader to know about the work of the parameters.